src/store/registration/saga.test.ts

```diff
@@ -457,6 +466,28 @@ describe('channelsLoaded', () => {
   });
 });
 
+describe('openInviteToastWhenRewardsPopupClosed', () => {
+  it('sets the invite open flag after the rewards propup is closed', async () => {
```

Comment:
Nice! I'll need to dive into some more of the tests here to fully understand testing the sagas but from a high level I can follow whats going on and makes sense